kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -141,6 +142,12 @@ export class FormWizard extends HTMLElement {
     this.#updateDoodle();
   }
 
+  handleEvent(event) {
+    if (event.type == "DeviceMigrationWizard:OpenReminderDialog") {
+      this.openReminderDialog();
+    }
+  }
+
```

Comment:
I think we'll need to move this to the setup-device step as well since `openReminderDialog` is no longer defined on the form-wizard

Reply:
Ah, I'll just go ahead and remove this altogether. In mozilla/sumo#1505, I'll wire this up to the button there.